### ISSUE TITLE
Add drivers for Windows ARM64

### ISSUE THREAD

**Issue by WeeJeWel** [raspberrypi/usbboot#148]
Currently, `BCM2711` does not seem to have a driver for Windows on Arm.

**Comment by WeeJeWel** [comment]
Why not? It's the future! :)

**Comment by timg236** [comment]
It's open source, you are free to fork it and roll it yourself. I can think of many more interesting things to do than port this code to every OS. 
It works on Raspberry Pi OS on Raspberry Pi which is good enough

**Comment by WeeJeWel** [comment]
Are we talking about the same thing? I mean https://github.com/raspberrypi/usbboot/blob/master/win32/rpiboot_setup.exe.

**Comment by timg236** [comment]
Yes. This isn’t going to get ported to ARM and isn’t really supported on win32 either

**Comment by WeeJeWel** [comment]
Ah, okay.
